api/capi/doc/nnstreamer_capi.md

```diff
@@ -0,0 +1,315 @@
+# nnstreamer capi
+
+
+An application which require a data stream pipeline with deep learning model filter can be constructed and assembled easily with nnstreamer c-api.
+
+The main features of nnstreamer-capi include:
+
+- **Construction of data pipeline based on [gstreamer](https://gstreamer.freedesktop.org/)**
+
+  You can compose the data stream pipeline through nnstreamer-capi with various elements which included at gstreamer.
+
+- **[Single-shot](#singleshotapi) & [Pipeline](#pipelineapi) API**
+  
+  There are 2 types of nnstreamer-capi. Single-shot API is useful to get the output result of the model file without the composition of the data pipeline.
+  Otherwise, Pipeline API allows developers to manipulate and compose pipelines with various [elements](#elementapi).
+  
+- **Support various Neural net frameworks (NNFW)**
+  
+  Tensorflow and Tensorflow-Lite are supported. It means the model files, trained through Tensorflow or Tensorflow-lite, can be imported as a filter(element) of a pipeline.
+  Moreover, the custom filter, trained in any kind of way, can be applied as a shared library.
```

Comment:
TensorFlow, TensorFlow-Lite, Caffe2, and PyTorch are support. Thus, neural network model files trained by such frameworks can be imported as filters of pipelines directly.
Custom filters, which are neural network models implemented directly with programming languages including C/C++ and Python, may be imported as filters of pipelines directly as well.

Note that, in Tizen release, this is limited to TensorFlow-Lite only; we do not ship other frameworks for official embedded device OS images.

Reply:
it's applied